docs/source/releases/latest/add-numpy-standards-502.yaml

```diff
@@ -0,0 +1,16 @@
+documentation:
+- description: |
+    Throughout GeoIPS, we have inconsistent coding standards. The numpy docstring format is an external standard we already use, and so we want that to be explicit in our documentation. This adds explicit info on using numpy docstrings, as well as examples.
```

Comment:
Shouldn't this also fail the RST linting?  Didn't we have a 120 character line length limit for RST?

Reply:
Since it's a yaml file, it's not checked :/ We can implement a check for it though - eg. running doc8 on the generated rst or something manual on the yaml

Reply:
I went ahead and shortened it by hand

Reply:
Oh, duh.  It's YAML....

I guess it doesn't matter - these are auto-generated into RST anyway, and short lived.